### PR TITLE
Set up technical docs folder

### PR DESCRIPTION
**Pull request by sdaniel55**
- Created `tech-docs` folder
- Created folders to organise documents
- Added Danger docs
- Moved beta testing docs from private repo

With 💚

### CODE REVIEW

File: tech-docs/testing/beta-testing.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## Development builds
+
+These builds are created after each PR is merged, making sure the most recent changes are available to test. These builds are not meant for production use, and are configurable for QA environments
+
+### How can I download a development build?
+
+If you need to test a feature that hasn't been released yet, follow these steps:
+
+1. Go to the `#chip-apps` Slack channel on your device
```
Comment (adamwaite) [inline tech-docs/testing/beta-testing.md:16]: Perhaps we should add a bit about `fastlane add_device` as a prerequisite to this.

File: tech-docs/testing/beta-testing.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+2. Find the latest iOS build (under the `App` title, it should say `Chip-iOS`)
+3. Tap the `Install Page` button
+4. Make sure the website is opened in Safari, not the in app browser
```
Comment (adamwaite) [inline tech-docs/testing/beta-testing.md:19]: Is this a thing? Not experienced this myself.

File: tech-docs/testing/beta-testing.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+5. Tap `Install`
+6. The app should appear on your home screen _(Note: this will be a different app from the App Store version)_
+
```
Comment (adamwaite) [inline tech-docs/testing/beta-testing.md:22]: Maybe add a bit about shake to access the debug menu?

**Review by adamwaite — APPROVED** [review]
Very nice work

File: tech-docs/testing/beta-testing.md (hunk 2 above)
Comment (sdaniel55) [reply tech-docs/testing/beta-testing.md:19]: IIRC I had an issue when I opened it inside Slack, but worked fine in Safari

File: tech-docs/testing/beta-testing.md (hunk 3 above)
Comment (sdaniel55) [reply tech-docs/testing/beta-testing.md:31]: 926b83b

File: tech-docs/testing/beta-testing.md (hunk 1 above)
Comment (sdaniel55) [reply tech-docs/testing/beta-testing.md:25]: 926b83b

**Pull request merged**